lib/spellNumber.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * A function which returns a true when
+ * a given array or object is empty
+ * @param {Array or Object} type
+ * @returns {boolean}
+ *
+ * Examples:
+ *  spellNumber('123') returns one hundred twenty three
```

Comment:
This might be a bit difficult to write thorough tests because there are so many possible inputs. 😢

Some examples:
```
spellNumber(123123) returns "one-hundred twenty-three thousand, one-hundred twenty-three"
spellNumber(123123123) returns some really long string
spellNumber('123123123123123123123123123123123123123123') returns 😱
```

The last example is possible because strings are not limited by integers. It might be a good idea to limit the input to just numbers. However, numbers can also be decimal values like 1.23 in JavaScript.

Reply:
Working on it.